### PR TITLE
Middleware: Deleting scenarios for rhdm and rhpam: dm7-dmn-introducti…

### PR DESCRIPTION
**Pull request by kmacedovarela**
Deleting scenarios for rhdm and rhpam: dm7-dmn-introduction, brms-loan-application, dm7-loan-application.
Removing images from brms, commenting rhdm/rhpam images.

@DuncanDoyle is there any other repos or references that should also be treated in order to delete these scenarios?

### CODE REVIEW

**Comment by kmacedovarela** [comment]
@DuncanDoyle done

**Comment by jankleinert** [comment]
@kmacedovarela @DuncanDoyle If these scenarios are being deleted, please also remove them  from the middleware pathway file: https://github.com/openshift-labs/learn-katacoda/blob/master/middleware-pathway.json

Do you need redirects put in place for any of these? For example if someone had one of those scenarios bookmarked or linked from another source, do you want it to point to a different scenario, or just let it 404 if there's not a replacement or another relevant scenario.

@jdob FYI in case the removal of these has any impact on your category restructuring

**Comment by kmacedovarela** [comment]
Thanks for the tip @jankleinert I missed that one. We won't need any redirects there, thanks.

File: middleware-pathway.json [hunk 1]
```diff
@@ -53,16 +53,6 @@
       "course_id": "brms-loan-application",
       "title": "JBoss BRMS Loan Application demo"
     },
-    {
```
Comment (jankleinert) [inline middleware-pathway.json:56]: The JBoss BRMS Loan Application demo needs to be removed too, correct?

**Comment by kmacedovarela** [comment]
@jankleinert missed that one, thx. It's done.

**Pull request merged**